.github/workflows/build.yaml

```diff
@@ -1,6 +1,9 @@
 name: Dart
 
 on:
+  schedule:
+    # “At 00:00 (UTC) on Sunday.”
+    - cron: '0 0 * * 0'
```

Comment:
👍 thu

Reply:
thu?

Reply:
Hmm, not sure how those extra chars got added -

Reply:
🤷